package.json

```diff
@@ -10,7 +10,7 @@
     "coverage": "jest --coverage",
     "debug": "yarn clean && yarn install && yarn build && yarn package",
     "package": "ncc build --source-map --license licenses.txt",
-    "release": "yarn clean && yarn install --production && yarn build && yarn package",
```

Comment:
Sorry, could we revert this. This is intentional as it weeds out dev dependencies. 

If this isn't working for you you might need to install ncc as described here - https://docs.github.com/en/actions/sharing-automations/creating-actions/creating-a-javascript-action#commit-tag-and-push-your-action